src/react-components/hub-create-panel.js

```diff
@@ -214,6 +226,22 @@ class HubCreatePanel extends Component {
                             <span>{environmentAuthor.name}</span>
                           </span>
                         ))}
+                      {environmentAuthor &&
```

Comment:
are there intentions to show multiple authors?

I ask because [_Medieval Fantasy Book_ has two authors: Jim and Pixel](https://github.com/MozillaReality/mr-social-assets/blob/eac28ed/Environments/MedievalFantasyBook/MedievalFantasyBook.bundle.config.json#L8-L20), but only Jim's name is shown in the card.

this is confusing because at the footer of the page I see `Medieval Fantasy Book by Pixel`

<img width="1283" alt="screen shot 2018-04-25 at 4 59 32 pm" src="https://user-images.githubusercontent.com/203725/39278844-3bb0a208-48aa-11e8-8943-035b3378a084.png">

two possible (interim) solutions come to mind:

1. swap the order in the [`authors` in the environment's JSON file](https://github.com/MozillaReality/mr-social-assets/blob/eac28ed/Environments/MedievalFantasyBook/MedievalFantasyBook.bundle.config.json#L8-L20)
2. show both authors and their org info. (requiring org names to be in parentheses or some other styling of the `by` line)